tests/Operation/CountDocumentsFunctionalTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace MongoDB\Tests\Operation;
+
+use MongoDB\Operation\CountDocuments;
+use MongoDB\Operation\InsertMany;
+use stdClass;
+
+class CountDocumentsFunctionalTest extends FunctionalTestCase
+{
+    public function testEmptyCollection()
+    {
+        $operation = new CountDocuments($this->getDatabaseName(), $this->getCollectionName(), []);
+        $this->assertEquals(0, $operation->execute($this->getPrimaryServer()));
```

Comment:
When possible, `assertSame()` is helpful to ensure we return the correct types. This is also nice because we can't yet you return type hints.

Reply:
I've added a separate commit to update the other test too

Reply:
Did you forget to push changes?